app/error/logger.js

```diff
@@ -0,0 +1,36 @@
+const { createLogger, format, transports } = require('winston');
```

Comment:
Just a comment about the location of this `logger` file. Could it be moved somewhere else?

Reply:
yes I just put it there for now so everyone could take a look and get feedback - it isn't the polished product!

Reply:
can we move this to the root for now. i think we are good with what you've done here.